v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/transforms/io/GcsIO.java

```diff
@@ -61,16 +63,27 @@
     /** Necessary {@link PipelineOptions} options for Pipelines that operate with JSON/CSV data in GCS. */
     public interface GcsPipelineOptions extends PipelineOptions {
         @Description("GCS filepattern for files in bucket to read data from")
-        String getInputGcsFilePattern();
+        String getInputGcsPath();
```

Comment:
Why did you decide to replace `FilePattern` with `Path`? It is called `inputGcsFilePattern` because we support wildcard patterns like `gs://<bucket>/<folder>/*.json`.

Reply:
Cause it is actually a path,  we don't do any patterns matching ans so on, it is a path to gcs file